packages/nbd/nbd.6.0.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "Xapi project maintainers"
+authors: [ "Jonathan Ludlam" "David Scott" "Thomas Sanders" ]
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/xapi-project/nbd"
+doc: "https://xapi-project.github.io/nbd/nbd/index.html"
+dev-repo: "git+https://github.com/xapi-project/nbd.git"
+bug-reports: "https://github.com/xapi-project/nbd/issues"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "-p" name "-j" jobs "@doc"] {with-doc}
+  ["dune" "build" "-p" name "-j" jobs "@runtest"] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "bisect_ppx" {dev & >= "2.5.0"}
+  "dune" {>= "2.7.0"}
+  "alcotest" {with-test}
+  "alcotest-lwt" {with-test}
+  "cstruct" {>= "6.0.0"}
+  "io-page"
+  "io-page-unix" {with-test}
+  "mirage-block" {>= "2.0.0"}
+  "mirage-block-unix"
+  "lwt" {>= "2.7.0"}
+  "lwt_log"
+  "odoc" {with-doc}
+  "ppx_cstruct" {>= "3.1.0"}
+  "ppx_sexp_conv" {>= "v0.9.0"}
+  "rresult"
+  "sexplib"
+  "uri"
+]
+tags: [ "org:mirage" "org:xapi-project" ]
```

Comment:
```suggestion
conflicts: ["result" {< "1.5"}]
tags: [ "org:mirage" "org:xapi-project" ]
```
Due to
```

#=== ERROR while compiling nbd.6.0.0 ==========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///home/opam/opam-repository
# path                 ~/.opam/4.08/.opam-switch/build/nbd.6.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p nbd -j 31
# exit-code            1
# env-file             ~/.opam/log/nbd-8-b09eba.env
# output-file          ~/.opam/log/nbd-8-b09eba.out
### output ###
# File "/home/opam/.opam/4.08/lib/io-page/io-page.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/lwt_log/lwt_log.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/lwt_log/core/lwt_log.core.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/re/re.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/re/emacs/re.emacs.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/re/glob/re.glob.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/re/pcre/re.pcre.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/re/perl/re.perl.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/re/posix/re.posix.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/re/str/re.str.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ocaml-compiler-libs/bytecomp/ocaml-compiler-libs.bytecomp.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ocaml-compiler-libs/common/ocaml-compiler-libs.common.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ocaml-compiler-libs/shadow/ocaml-compiler-libs.shadow.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ocaml-compiler-libs/toplevel/ocaml-compiler-libs.toplevel.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ppx_derivers/ppx_derivers.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/io-page-unix/io-page-unix.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
#       ocamlc lib/.nbd.objs/byte/nbd__Mux.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -w -34-32 -g -bin-annot -I lib/.nbd.objs/byte -I /home/opam/.opam/4.08/lib/base/caml -I /home/opam/.opam/4.08/lib/bigarray-compat -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/cstruct -I /home/opam/.opam/4.08/lib/cstruct-lwt -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/io-page -I /home/opam/.opam/4.08/lib/logs -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/lwt/unix -I /home/opam/.opam/4.08/lib/lwt_log -I /home/opam/.opam/4.08/lib/lwt_log/core -I /home/opam/.opam/4.08/lib/mirage-block -I /home/opam/.opam/4.08/lib/mirage-block-unix -I /home/opam/.opam/4.08/lib/mirage-device -I /home/opam/.opam/4.08/lib/mmap -I /home/opam/.opam/4.08/lib/ocaml/threads -I /home/opam/.opam/4.08/lib/parsexp -I /home/opam/.opam/4.08/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.08/lib/re -I /home/opam/.opam/4.08/lib/re/posix -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/rresult -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/sexplib -I /home/opam/.opam/4.08/lib/sexplib0 -I /home/opam/.opam/4.08/lib/stdlib-shims -I /home/opam/.opam/4.08/lib/stringext -I /home/opam/.opam/4.08/lib/uri -no-alias-deps -open Nbd -o lib/.nbd.objs/byte/nbd__Mux.cmo -c -impl lib/mux.pp.ml)
# File "lib/mux.ml", line 68, characters 32-40:
# 68 |     -> (unit, Protocol.Error.t) Result.t Lwt.t
#                                      ^^^^^^^^
# Error: Unbound type constructor Result.t
```